.github/workflows/validation.yaml

```diff
@@ -0,0 +1,57 @@
+name: Validations
+
+on:
+  pull_request:
+    types: [ opened, edited, synchronize ]
+  push:
+    branches:
+      - main
```

Comment:
Useless trigger, git diff will be empty.

Reply:
Fixed